sdk/nodejs/package.json

```diff
@@ -1,15 +1,18 @@
 {
-    "name": "@devcycle/nodejs-server-sdk",
-    "version": "1.15.4",
-    "description": "The DevCycle NodeJS Server SDK used for feature management.",
-    "license": "MIT",
-    "devDependencies": {
-        "@devcycle/bucketing-test-data": "^1.2.21"
-    },
-    "dependencies": {
-        "@devcycle/bucketing-assembly-script": "^1.8.3"
-    },
-    "engines": {
-        "node": ">=14.0.0"
-    }
+  "name": "@devcycle/nodejs-server-sdk",
+  "version": "1.15.4",
+  "description": "The DevCycle NodeJS Server SDK used for feature management.",
+  "license": "MIT",
+  "devDependencies": {
+    "@devcycle/bucketing-test-data": "^1.2.21"
+  },
+  "dependencies": {
+    "@devcycle/bucketing-assembly-script": "^1.8.3",
```

Comment:
Due to the changed to use the `external` option as part of the build, we now need to manually define our dependencies for when this will be published to NPM.